module_utils/ansible_nailgun_cement.py

```diff
@@ -224,6 +224,11 @@ def delete_entity(entity, module):
     return True
 
 
+def find_content_views(module, content_views, organization, failsafe=False):
+    return map(lambda content_view: find_content_view(module, content_view['name'], content_view['version'],
+                                                      organization, failsafe=failsafe), content_views)
+
+
```

Comment:
I think, this is unrelated. It should be in some 'refactor_content_view' PR.

Reply:
doh.. i meant to leave that out. thanks